scripts/add-cluster.sh

```diff
@@ -339,13 +306,6 @@ login_to_cluster ${JOINING_CLUSTER_TYPE}
 if [[ ${JOINING_CLUSTER_TYPE_NAME} != "e2e" ]]; then
     SA_NAME="toolchaincluster-${JOINING_CLUSTER_TYPE_NAME}${MULTI_MEMBER}"
     create_service_account
```

Comment:
you removed the second part of the if-statement, so you need to adjust also the first part
```suggestion
SA_NAME="toolchaincluster-${JOINING_CLUSTER_TYPE_NAME}${MULTI_MEMBER}"
create_service_account
```

Reply:
yea.. a silly miss, that if statement is not required now . Thanks for pointing it out